### DISCUSSION TITLE
arm架构 边缘Iot环境,  轻量模型4MB不到，服务端直接破G? 说好的轻量呢...

### DISCUSSION THREAD

**Discussion by gowy222** [PaddlePaddle/PaddleOCR#12941] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：arm32/64 物联网设备(没啥高配置)

-----
有大佬实现轻量化的解决方案么？有没有2GB边缘设备能跑ocr预测？

docker pull paddlepaddle/fastdeploy 4GB? 实际大概12GB...直接劝退
docker pull paddlepaddle/serving 1.44GB 实际大概需要4GB

没有1GB一下的CPU部署方案？求教..

**Comment by Gu-f** [comment]
没有了，目前这是最小的了

**Comment by zyscodes** [comment]
arm架构部署时有遇到这个问题吗？大佬
![image](https://github.com/user-attachments/assets/25aa5106-3a64-4c82-8d04-c50f95f09ecb)
